packages/apps/graph-client/src/components/extended-transfers-table/extended-transfers-table.tsx

```diff
@@ -0,0 +1,250 @@
+                {option}
+              </option>
+            ))}
+          </Select>
+        </div>
+        <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+          <Pagination
+            totalPages={totalPages}
+            label="pagination"
+            currentPage={currentPage}
+            onPageChange={handlePaginationClick}
+          />
+        </div>
+      </Box>
+      <Table.Root wordBreak="break-word">
+        <Table.Head>
+          <Table.Tr>
+            <Table.Th>Chain</Table.Th>
+            <Table.Th>Block Height</Table.Th>
+            <Table.Th>Amount</Table.Th>
+            <Table.Th>Sender Account</Table.Th>
+            <Table.Th>Receiver Account</Table.Th>
+            <Table.Th>Request key</Table.Th>
+          </Table.Tr>
+        </Table.Head>
+        <Table.Body>
+          {transfers.edges.map((edge, index) => {
+            /**  These transfers are going to be added to their crosschain counterpart and
+           this way we avoid repeated transfers in the table */
+            if (edge.node.transaction?.pactId) {
+              return <></>;
+            }
+
+            const chainIdDisplay = edge.node.crossChainTransfer
+              ? `${edge.node.chainId} / ${edge.node.crossChainTransfer.chainId}`
+              : edge.node.chainId;
+
+            const heightDisplay = edge.node.crossChainTransfer
+              ? `${edge.node.height} / ${edge.node.crossChainTransfer.height}`
+              : edge.node.height;
+
+            return (
+              <Table.Tr key={index}>
+                <Table.Td>{chainIdDisplay}</Table.Td>
+                <Table.Td>{heightDisplay}</Table.Td>
+                <Table.Td>{edge.node.amount}</Table.Td>
+                <Table.Td>
+                  <Link
+                    href={`${routes.ACCOUNT}/${router.query.module}/${edge.node.senderAccount}`}
+                  >
+                    {edge.node.senderAccount}
+                  </Link>
+                </Table.Td>
+                <Table.Td>
+                  {edge.node.receiverAccount ? (
+                    <Link
+                      href={`${routes.ACCOUNT}/${router.query.module}/${edge.node.receiverAccount}`}
+                    >
+                      {edge.node.receiverAccount}
+                    </Link>
+                  ) : edge.node.crossChainTransfer?.receiverAccount ? (
+                    <Link
+                      href={`${routes.ACCOUNT}/${router.query.module}/${edge.node.crossChainTransfer.receiverAccount}`}
+                    >
+                      {edge.node.crossChainTransfer.receiverAccount}
+                    </Link>
+                  ) : (
+                    <span style={{ color: 'lightgray' }}>N/A</span>
+                  )}
+                </Table.Td>
+                <Table.Td>
+                  <Link href={`${routes.TRANSACTIONS}/${edge.node.requestKey}`}>
+                    {edge.node.requestKey}
+                  </Link>
+                  /
```

Comment:
only add the / if there is a edge.node.crossChainTransfer

Reply:
Fixed